graphql/schemas/Guild/people.graphql

```diff
@@ -10,6 +10,10 @@ type People {
     dob: Date
     contacts: [Contact!]! @hasMany
     address: [Address!]! @hasMany
+    peoplesEmploymentHistory: [PeopleEmploymentHistory!]!
```

Comment:
change this to employment_history , since we are using _ on the properties

    employment_history: [PeopleEmploymentHistory!]!